src/views/Project/index.jsx

```diff
@@ -111,12 +136,17 @@ export default class Project extends Component {
       [];
 
     return (
-      <Fragment>
-        <header>
+      <div className={classes.root}>
+        <AppBar position="absolute" className={classes.header}>
+          <IconButton className={classes.link} component={NavLink} to="/">
+            <ArrowLeftIcon />
+          </IconButton>
           <Typography variant="display2" align="center">
```

Comment:
I would try `variant="display1"` here. Also, make sure there the title and back button have equal space in the top and bottom (i.e, vertically aligned to the app bar).